inference-engine/src/transformations/src/transformations/softplus_decomposition.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/softplus_decomposition.hpp"
+
+#include <memory>
+#include <vector>
+
+#include <ngraph/opsets/opset4.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+ngraph::pass::SoftPlusDecomposition::SoftPlusDecomposition() {
+    auto input = ngraph::pattern::any_input();
+    auto softplus = std::make_shared<ngraph::opset4::SoftPlus>(input);
+
+    ngraph::graph_rewrite_callback callback = [=](ngraph::pattern::Matcher& m) {
+        auto & pattern_to_output = m.get_pattern_value_map();
+        auto softplus_input = pattern_to_output.at(input);
+
+        auto exp = std::make_shared<ngraph::opset4::Exp>(softplus_input);
```

Comment:
The decomposition must be executed for plugins which does not support SoftPlus. Please, add a call to the `m_transformations_callback` to check that.

Reply:
Check added.